devel/qt5-qtcreator/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           cxx11  1.1
 
 name                qt5-qtcreator
 
-version             4.6.0
+version             4.6.1
 #revision            0
```

Comment:
Why the commented out `revision`?

Reply:
There is no compelling reason for the comment.
Whenever someone has `revision  0`, it is correctly pointed out that this is the default and should not be there.
However, I find it a very very minor inconvenience to retype `revision` when I want to increase the revision.

Reply:
Normally we don't do this, but it's your port.